server/routes/contacts.js

```diff
@@ -1,14 +1,14 @@
-var express = require('express');
+var express = require("express");
 var router = express.Router();
 var controller = require("../controllers/Contacts");
 
-// Manage a single user's contacts
+// Manage a single user"s contacts
```

Comment:
this is an apostrophe not a single quote

Reply:
my bad, find and replace 😅

Reply:
Yh thought so